src/custom/bluerov.rs

```diff
@@ -69,15 +81,17 @@ pub fn rtsp() -> Vec<VideoAndStreamInformation> {
 
             // Get the biggest resolution possible
             let mut sizes = format.sizes.clone();
-            sizes.sort_by(|first_size, second_size| {
-                (10 * first_size.width + first_size.height)
-                    .cmp(&(10 * second_size.width + second_size.height))
-            });
+            sort_sizes(&mut sizes);
+
+            if sizes.last().is_none() {
+                warn!("Unable to find a valid size for {:?}", cam);
+                return None;
+            }
             let size = sizes.last().unwrap();
```

Comment:
this could be:
```suggestion
            let Some(size) = sizes.last() else {
                warn!("Unable to find a valid size for {cam:?}");
                return None;
            };
```